go/README.md

```diff
@@ -118,3 +118,57 @@ func main() {
     }
 }
 ```
+### Custom Context
+
+If you want even greater control over the lifecycle of the HTTP requests consider providing a [context](https://pkg.go.dev/context). Contexts can be set for all requests or per request. They can be combined with the timeout options mentioned in the previous section to fine-tune the lifecycle of your requests.
```

Comment:
Change to: "You can set a [context](https://pkg.go.dev/context) for all requests or a specific request for more control over the SDK's HTTP requests. "